CHANGES.rst

```diff
@@ -58,3 +58,6 @@ Bugs fixed
 
 Testing
 -------
+
+* #13224: Correctness fixup for ``test_html_multi_line_copyright``.
```

Comment:
I acknowledge that this entry may not be strictly required, but we do have a changelog section for `Testing`, and I think this fits within that topic.